src/bindings/python/CMakeLists.txt

```diff
@@ -108,6 +108,8 @@ set(SOURCES
 	PyViewTransform.cpp
 )
 
+# Reason not to use pybind11_add_module ?
+# https://pybind11.readthedocs.io/en/stable/compiling.html#pybind11-add-module
```

Comment:
If that's the right way to do it, I will support & suggest to make the change.

Reply:
I was mostly curious if this was a conscious choice or not, @michdolan do know more about this ?

I can make some investigations if needed.

Reply:
I was not aware of `pybind11_add_module`. I agree it seems like a good idea to use it. Thanks for pointing it out @remia